ci/jenkins/README.md

```diff
@@ -121,34 +123,6 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY} --proxyall
```

Comment:
I see a reference to `--proxyall` here, but I think you are removing the command-line flag as part of this PR?